manifests/params.pp

```diff
@@ -22,12 +22,12 @@
       }
     }
     'Debian': {
-      $default_su_user = versioncmp($facts['operatingsystemmajrelease'], '14.00') ? {
-        1        => 'root',
+      $default_su_user = $facts['operatingsystem'] ? {
```

Comment:
just as a note in case somebody checks this in the future: This change is fine because we do not support Ubuntu older than 14.04 (`versioncmp()` used `14.00`, not `14.04`)